### PR TITLE
fix managing freeswitch ipv6-profiles with command

### LINKED ISSUE #239: [BUG] - Role fails with bbb_freeswitch_ipv6: false after the role was run with bbb_freeswitch_ipv6: true
**Describe the bug**
Second run of role fails at manage ipv6 sip_profiles

**To Reproduce**
run role twice on the same machine with ipv6 disbled

**Expected behavior**
Running twice doesnt result in an error

**Screenshots**
DO NOT USE SCREENSHOTS TO SHOW CODE!
This makes the information inaccessible to visually impaired users.

**Desktop information if needed (please complete the following information):**
 - OS: ubuntu20.04
 - ansible 2.9.6
 - BBB 2.3.13

**Your variables for this role - be sure to remove any sensitive information**

```
failed: [xxx] (item=/opt/freeswitch/etc/freeswitch/sip_profiles/external-ipv6.xml.disabled) => {
    "ansible_loop_var": "sip_profile",
    "changed": false,
    "invocation": {
        "module_args": {
            "_raw_params": null,
            "_uses_shell": false,
            "argv": null,
            "chdir": null,
            "creates": null,
            "executable": null,
            "removes": null,
            "stdin": null,
            "stdin_add_newline": true,
            "strip_empty_ends": true,
            "warn": true
        }
    },
    "msg": "no command given",
    "rc": 256,
    "sip_profile": {
        "ansible_loop_var": "item",
        "changed": false,
        "failed": false,
        "invocation": {
            "module_args": {
                "checksum_algorithm": "sha1",
                "follow": false,
                "get_attributes": true,
                "get_checksum": true,
                "get_md5": false,
                "get_mime": true,
                "path": "/opt/freeswitch/etc/freeswitch/sip_profiles/external-ipv6.xml"
            }
        },
        "item": "/opt/freeswitch/etc/freeswitch/sip_profiles/external-ipv6.xml",
        "stat": {
            "atime": 1631295795.5290966,
            "attr_flags": "e",
            "attributes": [
                "extents"
            ],
            "block_size": 4096,
            "blocks": 16,
            "charset": "us-ascii",
            "checksum": "3d7db4d500f06c25769925bc7de547353dda7c45",
            "ctime": 1631286197.1041253,
            "dev": 2053,
            "device_type": 0,
            "executable": false,
            "exists": true,
            "gid": 1,
            "gr_name": "daemon",
            "inode": 5120716,
            "isblk": false,
            "ischr": false,
            "isdir": false,
            "isfifo": false,
            "isgid": false,
            "islnk": false,
            "isreg": true,
            "issock": false,
            "isuid": false,
            "mimetype": "text/plain",
            "mode": "0644",
            "mtime": 1622681453.0,
            "nlink": 1,
            "path": "/opt/freeswitch/etc/freeswitch/sip_profiles/external-ipv6.xml",
            "pw_name": "freeswitch",
            "readable": true,
            "rgrp": true,
            "roth": true,
            "rusr": true,
            "size": 5796,
            "uid": 999,
            "version": "1905633503",
            "wgrp": false,
            "woth": false,
            "writeable": true,
            "wusr": true,
            "xgrp": false,
            "xoth": false,
            "xusr": false
        }
    }
}
```

### PR DESCRIPTION
**Pull request by crosscodr**
closes #239

This is just a strange bug in ansible:
https://github.com/ansible/ansible/issues/72295
https://github.com/ansible/ansible/issues/71817

### CODE REVIEW

**Review by BrutalBirdie — APPROVED** [review]
Strange but OK.

File: tasks/freeswitch.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@
 
     - name: manage ipv6 sip_profiles
       become: true
-      ansible.builtin.command: |
+      command: >
```
Comment (EdJoPaTo) [inline tasks/freeswitch.yml:122]: Replacing `ansible.builtin.command` with `command` to satisfy the check… ok.

But replacing `>` with `|` should result in something not wanted according to my understanding of yaml.

`|` preserves newlines and `>` folds them into spaces which is kinda what I would expect from a `mv A B` command.

https://yaml-multiline.info/

I would expect to keep the `|`

```suggestion
      command: |
```

Comment (crosscodr) [reply tasks/freeswitch.yml:122]: Hi @EdJoPaTo,
this is exactly the reason, why I changed it to `>` :D
On the command line, you would separate arguments for `mv` with spaces. So the behavior of `>` better emulates this, in my understanding.
However, both versions do work.

Comment (EdJoPaTo) [reply tasks/freeswitch.yml:122]: Oh yeah, guess I confused myself, you are right :D

**Review by Davka — APPROVED** [review]

**Pull request merged**